changelog.txt

```diff
@@ -16,6 +16,7 @@ that repo.
 ## New Scripts
 - `exportlegends`: export extended legends information for external browsing
 - `modtools/create-item`: commandline and API interface for creating items
+- `light-aquifers-only`: convert heavy aquifers to light (available as an fort autostart option in `gui/control-panel`)
```

Comment:
Perhaps this could go under Improvements instead?

Actually, none of these are new, strictly speaking - they are making a return in v50, but it looks like light-aquifers-only in particular probably would have worked before this PR too. I flagged these as `reinstated` in the dfhack/dfhack changelog.txt, which I guess could work here too. Or we could add a new section.

I think the control-panel capability could stand as a separate improvement, either way.

Reply:
@ab9rf and I had this discussion a few weeks ago too. We were adding these to "New Scripts" since we thought it fit the intent of this section, as far as our current users are concerned. "reinstated" sounds appropriate too. I can add that in here.

Reply:
Presenting them to new users like this makes sense to me, but I didn't want to suggest that they were new to everyone. Glad `reinstated` sounds good